docs/install/plugin-restrictions.rst

```diff
@@ -0,0 +1,16 @@
+Known Limitations
+=================
+
+* You should not move a volume from one node to another unless you are sure no containers are using the volume.
+
+  The plugin will not stop volumes from being migrated out from underneath a running container.
+  It is possible that Docker or your orchestration tool will prevent this from happening, but Flocker itself does not.
+* ``--volumes-from`` and equivalent Docker API calls will only work if both containers are on the same machine.
```

Comment:
I don't know whether this is true, but I'll take your word for it.

Do these known limitations have JIRA bugs? They need not be referenced here, but it would be good if the JIRA bugs had notes saying "Update plugin-restrictions.rst when you fix". Otherwise, this risks getting out of date. (Or maybe you have another system for keeping on top of that sort of thing?)